src/ai/baseAi.ts

```diff
@@ -0,0 +1,16 @@
+import Ain from '@ainblockchain/ain-js';
+import Ainize from '@ainize-team/ainize-js';
+
+import ChatAi from './chatAi';
+
+export default class BaseAi {
+  chat: ChatAi;
```

Comment:
(minor) maybe this definition can be moved after ain and ainize definitions..

Reply:
Done!